portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,17 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <ul id="nav">
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="resume.html">Resume</a></li>
+        <li><a href="contact.html">Contact</a></li>
+    </ul>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1 class="title">Coco Sack's Portfolio</h1>
+    </div>
+    <div class="container">
+        <img class="titleImage" src = "/images/Coco2.png" alt="Coco Photo">
```

Comment:
Separate words in ID and class names by a hyphen: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters